src/CodeGen_Hexagon.cpp

```diff
@@ -1810,6 +1818,16 @@ void CodeGen_Hexagon::visit(const Call *op) {
                                 type_suffix(op->args[1], op->args[2], false),
                                 op->args);
             return;
+        } else if (op->is_intrinsic(Call::abs)) {
+            internal_assert(op->args.size() == 1);
+            Type ty = op->args[0].type();
+            if ((ty.is_vector() && ty.is_int())) {
+                if ((ty.bits() == 8 && (target.features_any_of({Target::HVX_v65, Target::HVX_v66})))
```

Comment:
I think it would be more future proof if this checked for targets that don't support abs.vb (a finite list), rather than ones that do (which is a list that will keep growing indefinitely)